templates/tutorials/index.html

```diff
@@ -4,124 +4,154 @@
 
 {% block meta_copydoc %}https://docs.google.com/document/d/1Lpo7Od9VA1aHYmr9UVp02vCGOP5Zxmzx4zZE9YJwlQg/edit{% endblock %}
 
+{% block meta_description %}Learn what you can do with MicroK8s and how, through a series of tutorials on Ubuntu, Windows and macOS using laptops, Raspberry Pis and edge devices.{% endblock %}
+
 {% block content %}
 
-<section class="p-strip">
```

Comment:
In the design it looks like the title is wrapped in a `strip` and then a `section`. Based on that I suppose the code should look something like this:
```
<section class="p-strip u-no-padding--bottom">
  <div class="p-section--deep">
    <div class="u-fixed-width">
      <h1>Tutorials</h1>
    </div>
  </div>
</section>
```

Feel free to ignore it, Visual might have a different opinion. I only noticed it because the title looked a little closer to the navbar than it did in the design.

Reply:
yes, we updated the documentation - nesting is very much needed. Please make the strip shallwo though - in dev mode you can see in the design it is meant to be shallow:

![image](https://github.com/canonical/microk8s.io/assets/2741678/b9a4467b-e7ee-45c1-9003-014a9a6fdc53)